README.md

```diff
@@ -1,24 +1,279 @@
+**URI Service** is a standalone Rails 5 application that creates/stores local and temporary terms and caches external URI terms. To start the application's only interface will be a JSON api, though we see expanding the application to include html pages for local terms.
 
-Things you may want to cover:
+URI Service makes some assumptions about how URIs should be organized. A URI is always related to a vocabulary. A vocabulary has many URI terms. Multiple authorities might be represented within a vocabulary. A URI must be unique to a vocabulary, but a URI can appear in multiple vocabularies.
 
-* Ruby version
+In order to use the JSON api, URI Service requires token authentication. URI Service does not provide granular authorization. The application that uses the api should control authorization in anyway it deems necessary.
 
-* System dependencies
+## Definitions
+#### External Term
+Used when caching a value/URI pair from an external controlled vocabulary within URI Service. Example: We want to add an entry for U.S. President Abraham Lincoln to our URI Service datastore, so we'll create an external term that references his Library of Congress URI.
 
-* Configuration
+#### Local Term
+Used when defining locally-managed terms in the URI Service. Automatically creates a local URI for a new local term. Example: We want to maintain a vocabulary for various departments within a university, and we want to create locally-managed URIs for these departments.
 
-* Database creation
+#### Temporary Term
+Used when you want to add a value to your URI Service datastore, but have no authority information about the term or do not wish to create a local URI. No two temporary terms within the same vocabulary can have the same value. Basically, a temporary term is intended to identify an exact string value rather than identifying an intellectual entity. Temporary terms should eventually be replaced by external or local terms later on when more information is known about the entity to which you are referring . Example: We want to record information about the author of an old and mysterious letter by "John Smith." We don't know which "John Smith," this refers to, so we'll create (or re-use) a temporary URI that's associated with the value "John Smith."
 
-* Database initialization
+#### Custom Fields
+TODO
 
-* How to run the test suite
+## Installation
+TODO
 
-* Services (job queues, cache servers, search engines, etc.)
+## Services (job queues, cache servers, search engines, etc.)
+URI Services stores local, temporary and external terms in a mysql database. Solr is used to index terms for fast lookup and searching. Redis/Resque will be used to run scheduled jobs.
 
-* Deployment instructions
+## Deployment instructions
+TODO
 
-* ...
+## API v1
+
+### Vocabularies
+#### List Vocabularies
+- **Request**
+  `GET /vocabularies`
+
+- **Success Response**
+
+  Sorts vocabularies alphabetically by label. Defaults to `page = 1` and `per_page = 20`.
```

Comment:
We should update the vocabularies response example below to have the terms sorted alphabetically by label.